website/screens/components/inline/code/examples/basicUsage.ts

```diff
@@ -0,0 +1,24 @@
+import { DxcInline } from "@dxc-technology/halstack-react";
+import Placeholder from "./Placeholder";
+
+const code = `() => {
+  return (
+    <DxcInline gutter="0.5rem" divider>
```

Comment:
What do you think about adding DxcInset to wrap the example?

From this:
![image](https://user-images.githubusercontent.com/34685461/177513385-9a766c05-30fc-4356-bd2a-c10b97039306.png)

To this:
![image](https://user-images.githubusercontent.com/34685461/177513424-95bd4cb9-b235-41f0-9c69-c4790c39d439.png)

Reply:
Yes, it is a good idea!